SPECIFICATION.md

```diff
@@ -224,13 +234,23 @@ parameters:
 2. Global letter spacing (`letter_spacing` from header)
 3. Leading space (leftmost X-coordinate) of second letter
 
+### Line Spacing
+
+Line spacing is the vertical distance between the lines of a multiline text. It
```

Comment:
Hm, that's not how line spacing is generally defined. Line spacing (aka leading) is usually baseline to baseline: https://en.wikipedia.org/wiki/Leading I'd stick with that.